README.md

```diff
@@ -25,7 +25,9 @@ end
 
 Then run `mix deps.get` on your terminal.
 
-Configure your application as seen in the *Configuration* section below prior to attempting to generate the migration or you will get an `application was not loaded/started` error.
+Configure your application as seen in the *Configuration* section below prior to
+attempting to generate the migration or you will get an
+`application was not loaded/started` error.
```

Comment:
Can we drop the changes to this file @yordis? We don't need to enforce a line limit in our markdown files.